soot-infoflow/src/soot/jimple/infoflow/data/SootFieldAndMethod.java

```diff
@@ -0,0 +1,137 @@
+package soot.jimple.infoflow.data;
+
+import soot.SootField;
+
+public class SootFieldAndMethod {
+
+    private final String fieldName;
+    private final String className;
+    private final String fieldType;
+
+    private Boolean isStatic;
```

Comment:
Why do you need this information here? You can't get it from the source/sink definition, and it doesn't help to distinguish fields. You can only get it from the Scene once you have resolved the field to a proper ``SootField``. I'd therefore leave it out here.

Reply:
Thanks for your suggestion, Steven. I deleted all the useless fields.